tools/update_rust_version.sh

```diff
@@ -4,13 +4,21 @@
 # Ask rustup to pick the latest version that will work.
 # This requires rustup >= 1.20.0.
 echo "Updating rustc to latest compatible version..."
-rustup update nightly >/dev/null 2>/dev/null
+rustup update nightly
 
-# Rerun the command so that it prints out the version it installed. We then have
-# to extract that from the output. If there is a better way to do this then we
-# should update this.
-RUSTUP_NIGHTLY_VERSION=`rustup update nightly 2>/dev/null`
-BEST_DATE=`echo $RUSTUP_NIGHTLY_VERSION | sed 's/.* \([0-9]*-[0-9]*-[0-9]*\).*/\1/g'`
+# # Rerun the command so that it prints out the version it installed. We then have
+# # to extract that from the output. If there is a better way to do this then we
+# # should update this.
+# RUSTUP_NIGHTLY_VERSION=`rustup update nightly 2>&1`
+# BEST_DATE=`echo $RUSTUP_NIGHTLY_VERSION | sed 's/.* \([0-9]*-[0-9]*-[0-9]*\).*/\1/g'`
+
+# I just do not know how to get rustup to tell us the version of the toolchain
+# it decided on with the format required for `rust-toolchain`. That the dates
+# are off-by-one day is annoying. I'm resorting to just asking the user.
+
+echo "Please enter the version of Rust to use."
+echo "It is probably just one day later than whatever was printed out above."
+read BEST_DATE
```

Comment:
This is unfortunate, but really probably only run by Tock core folks, so not really a problem